db/createItem.go

```diff
@@ -41,6 +41,7 @@ func (d *dynamoDB) CreateItem(listID string, name string) (*data.Item, error) {
 		if e.Code() == dynamodb.ErrCodeConditionalCheckFailedException {
 			return nil, ErrorIDExists
 		}
+		return nil, err
```

Comment:
Interesting, I would have assumed this would have fallen through to the `default` case without a break.

Reply:
Don't think so
https://play.golang.org/p/QG-F1ohcuc9